atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -37,8 +37,12 @@ def __init__(self, workflow, ignore_autogenerated_gpg_keys=True):
         super(PostBuildRPMqaPlugin, self).__init__(workflow)
         self.ignore_autogenerated_gpg_keys = ignore_autogenerated_gpg_keys
 
-    def run(self):
-        return self.workflow.build_dir.for_each_platform(self.gather_output)
+    def run(self) -> None:
+        if self.workflow.data.image_components:
+            self.log.info('skipping since flatpak_create_oci has already set rpm components')
```

Comment:
I would suggest a more generic message, something like the comment we had in OSBS 1
`# If another component has already filled in the image component list, skip`

Reply:
updated.